ArjunaCore/pom.xml

```diff
@@ -28,6 +28,10 @@
   <packaging>pom</packaging>
   <name>JBoss Core Examples</name>
   <description>All quickstarts of JBoss Core</description>
+  <properties>
+    <maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
Could you, please, remind me why it's needed to move the compilation source and target for Java 1.6? From the high level perspective we should aim for java 8. But there is some reason for sure but I don't remember what is it.

Reply:
Its just for maven to use minimum compilation source and target for Java as 1.6, I guess by default maven uses java 1.5. But okay I will change it to 1.8, that would be better